openapi/platform/platform.yaml

```diff
@@ -708,12 +708,12 @@ paths:
       tags:
         - options
         - partners
-  /wholesaleInvoices:
+  /payableInvoices:
```

Comment:
Should we take this a step further and go with `accountsPayableInvoices`?

Reply:
That makes sense to me. I guess there are just some open questions that remain around the naming on the RFC, so I'll try to resolve those.

Reply:
Is there an advantage to a longer name? You could just update the description for the model. 
As [mentioned in the RFC](https://vendasta.jira.com/wiki/spaces/RD/pages/825327688/Exposing+Invoices+to+Partners+via+API?focusedCommentId=851181883#comment-851181883), I'm not sure if having the customer id on each line item makes sense for a generic payable invoice but I'll leave the decision up to the @vendasta/psycho-billers.

Reply:
I think the benefit of AccountsPayable vs Payable is that AccountsPayable is a more common term in accounting (for example, in [QuickBooks](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/apagingdetail), although they say AP & AR). That being said, it's a different domain than what is with ChargeBee, so I'm not an expert here.